better-font-awesome.php

```diff
@@ -361,11 +361,20 @@ public function create_admin_page() {
 			<form method="post" action="options.php" id="bfa-settings-form">
 			<?php
 				// This prints out all hidden setting fields.
-				settings_fields( 'option_group' );
+				settings_fields( self::SLUG );
```

Comment:
Won't changing this value cause existing saved settings/options to disappear? What's the motivation for changing this value?

Reply:
If we had been using the settings the WordPress way, that would be true, because WordPress would save the settings in the _wp_options_ table as **option_group_options** (a very "general" name that appears in a lot of tutorials, so, other plugins could overwrite our settings).

But right now this part of the code is being ignored, because the form is never submitted, there is no submit button, we have a <span> looking as a button. Instead of sending the form and let WordPress to save the settings, we are sending an Ajax call and saving the options ourselves with the options name: better-font-awesome_options (that is what WordPress would do if we use settings_fields( self::SLUG ). So, I'm trying to make our custom saving process to match what WordPress would do if we remove Ajax.